src/tools/annotation/ArrowAnnotateTool.js

```diff
@@ -27,8 +27,8 @@ import { textBoxWidth } from './../../drawing/drawTextBox.js';
  * @extends Tools.Base.BaseAnnotationTool
  */
 export default class ArrowAnnotateTool extends BaseAnnotationTool {
-  constructor(configuration = {}) {
-    const defaultConfig = {
+  constructor(props = {}) {
+    const defaultProps = {
```

Comment:
This naming is better, but we may want to use something a bit more verbose to reduce confusion. I'll noodle on it.

Reply:
I will be happy to find out a better name :)